### PR TITLE
Allows lower case variable names

### LINKED ISSUE #3: Error with parsing lower case variable names
I have variable names in the format of "export TF_VAR_foo=foobar" and they are being ignored  by the parser.

I believe the issue is here:

```
identifier :: Parser String
identifier = do
    x <- upper <|> underscore
    ys <- many $ upper <|> digit <|> underscore

    pure (x:ys)

  where
    underscore = char '_'
```

To reproduce you can run this in ghci:

```
λ: parse parseEnvironment "" "export TF_VAR_foo=foo\n"
Right []
```

### PR DESCRIPTION
**Pull request by denibertovic**
As per updated spec: http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap08.html

Closes #3

### CODE REVIEW

File: src/LoadEnv/Parse.hs [hunk 1]
```diff
@@ -42,12 +42,12 @@ parseVariable = do
 -- and the '_' (underscore) from the characters defined in Portable Character
 -- Set and do not begin with a digit.
 --
--- <http://pubs.opengroup.org/onlinepubs/000095399/basedefs/xbd_chap08.html>
+-- <http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap08.html>
```
Comment (pbrisbin) [inline src/LoadEnv/Parse.hs:51]: Can you update the paragraph above as well, to quote from the updated spec?

File: test/LoadEnv/ParseSpec.hs [hunk 2]
```diff
@@ -97,13 +97,16 @@ spec = do
                 `shouldBe` Right ("S3_KEY", "abc123")
             parse parseVariable "" "_S3_KEY=abc123\n"
                 `shouldBe` Right ("_S3_KEY", "abc123")
+            parse parseVariable "" "S3_key=abc123\n"
+                `shouldBe` Right ("S3_key", "abc123")
+            parse parseVariable "" "s3_key=abc123\n"
+                `shouldBe` Right ("s3_key", "abc123")
+
```
Comment (pbrisbin) [inline test/LoadEnv/ParseSpec.hs:104]: Please remove this extra line of white space.

**Review by pbrisbin — CHANGES_REQUESTED** [review]
Awesome! I requested some minor changes, but the substance of it looks great.

File: src/LoadEnv/Parse.hs (hunk 1 above)
Comment (denibertovic) [reply src/LoadEnv/Parse.hs:51]: :+1:

File: test/LoadEnv/ParseSpec.hs (hunk 2 above)
Comment (denibertovic) [reply test/LoadEnv/ParseSpec.hs:104]: :+1:

**Comment by denibertovic** [comment]
@pbrisbin I've addressed all of the request changes (I think). Let me know if you want me to make other changes.

**Review by pbrisbin — APPROVED** [review]

**Comment by pbrisbin** [comment]
Thank you! Will release soon.

**Pull request merged**